editor/translations/pot_generator.cpp

```diff
@@ -79,6 +79,8 @@ void POTGenerator::generate_pot(const String &p_file) {
 			return;
 		}
 
+		EditorTranslationParser::get_singleton()->customize_strings(translations);
```

Comment:
The doc says that `_customize_strings()` is called after parsing all files, but it's actually called after parsing each file here.

Reply:
Ugh, that's a problem, because I don't see easy way to put all translations into single vector. And calling the method for each file will require some ugly workarounds if you want to add new strings.

Reply:
Fixed it, but it makes generation less efficient :/